meson.build

```diff
@@ -30,8 +30,11 @@ add_project_arguments(cxx.get_supported_arguments(extra_flags), language: 'cpp')
 
 conf = configuration_data()
 conf.set_quoted('PACKAGE_VERSION', meson.project_version())
+conf.set_quoted('DATADIR', join_paths(get_option('prefix'), get_option('datadir')))
 config_h = configure_file(output : 'config.h', configuration : conf)
 
+meson.add_devenv({'NETPANZER_DATADIR': meson.project_source_root()})
```

Comment:
@eli-schwartz Strange, it worked on only 2 out of 4 Linux tests... https://github.com/netpanzer/netpanzer/actions/runs/7522200856/job/20473915895#step:9:17

Reply:
@eli-schwartz oops, my mistake. It was the 2 scons tests that passed. I'll have to take another look and see what I did wrong with the meson setup.